src/xunit.netcore.extensions/xunit.netcore.extensions.csproj

```diff
@@ -10,6 +10,8 @@
     <RootNamespace>Xunit.NetCore.Extensions</RootNamespace>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
```

Comment:
Since xunit-rc1 is built as a v5.0, I concluded that these extensions will have to specific that they are v5.0 as well. I could try removing it and rebuilding and seeing it still works with CoreFX, though.